github/ci/prow/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits-0.31.yaml

```diff
@@ -0,0 +1,96 @@
+---
+presubmits:
+  nmstate/kubernetes-nmstate:
+    - name: pull-kubernetes-nmstate-unit-test-0.31
+      branches:
+        - release-0.31
+      annotations:
+        fork-per-release: "true"
```

Comment:
This is relevant only on master. We don't want to fork stable branch on another release.